Audacia.Log.AspNetCore/Audacia.Log.AspNetCore.csproj

```diff
@@ -8,7 +8,7 @@
     <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
     <Copyright>Copyright © Audacia $([System.DateTime]::Now.Year)</Copyright>
     <PackageIcon>package-icon.png</PackageIcon>
-    <Version>3.0.0</Version>
+    <Version>3.1.0</Version>
```

Comment:
As `AddRequestBodyTelemetry` has been renamed to `AddActionRequestBodyTelemetry` should this actually bump to v4.0.0 as this is a breaking change?

Reply:
Yeah that's valid, can up to be on version 4.

Reply:
Bumped.